qml/plugins/RestartBluetooth.qml

```diff
@@ -0,0 +1,17 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.Utilities 1.0
+
+ActionItem {
+    //% "Bluetooth"
+    title: qsTrId("settings_bluetooth-bluetooth")
+    //% "Restart"
+    actionName: qsTrId("sailfish-tools-bt-restart")
+    deviceLockRequired: false
+    //% "Restart Bluetooth subsystem"
```

Comment:
```suggestion
    //% "Restart Bluetooth subsystem, which may help if you're "
    //% "having trouble scanning or connecting to a Bluetooth "
    //% "device which worked previously."
```
I'm thinking it might be good to fill this out a bit. What do you think?